src/tests/trainers.test.js

```diff
@@ -1,3 +1,69 @@
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+import request from 'supertest';
+import app from '../app';
+import Trainer from '../models/Trainer';
+import trainerSeed from '../seeds/trainers';
+
+const mockTrainerId = '64667332ecb50c522415bea5';
+
+const mockTrainer = {
+  firstName: 'Galo',
+  lastName: 'Durante',
+  dni: '37966681',
+  phone: '11457345911',
+  email: 'lolero@gmail.com',
+  password: 'askda2314K',
+  city: 'pepitojuarez',
+  salary: 2355635,
+  isActive: true,
+};
+
+beforeAll(async () => {
+  await Trainer.collection.insertMany(trainerSeed);
+});
+
+describe('PUT /api/trainers', () => {
+  test('check if the data change correctly', async () => {
+    let response = await request(app).get(`/api/trainers/${mockTrainerId}`).send();
+    const seedGet = response.body.data;
+    response = await request(app).put(`/api/trainers/${mockTrainerId}`).send(mockTrainer);
+    expect(response.status).toBe(200);
+    response = await request(app).get(`/api/trainers/${mockTrainerId}`).send();
+    expect(seedGet !== response.body.data).toBeTruthy();
```

Comment:
im not sure about this line comparing the entire objects maybe you could compare an important key/value from both